src/pages/HomePage.tsx

```diff
@@ -1,27 +1,29 @@
-import { Heading } from "@chakra-ui/react";
 import React from "react";
-import { User, usersApi } from "@/utils/fetcher";
+import { Heading, Text } from "@chakra-ui/react";
+import { Movie, moviesApi } from "@/utils/fetcher";
 
 const HomePage = () => {
   const [loading, setLoading] = React.useState(false);
-  const [users, setUsers] = React.useState<User[]>([]);
+  const [movies, setMovies] = React.useState<Movie[]>();
 
   React.useEffect(() => {
     setLoading(true);
-    usersApi.getUsers().then((users) => {
-      setUsers(users);
-      setLoading(false);
-    });
+    moviesApi
+      .getMovies({ apiKey: "11dfe233fe073aab1aaa3389310e3358" })
+      .then((res) => {
+        setMovies(res.results);
+        setLoading(false);
+      });
   }, []);
 
   return (
     <div>
       {loading ? (
         <p>loading...</p>
       ) : (
-        users.map((user) => (
-          <Heading as={"h1"} key={user.id}>
-            {user.name}
+        movies?.map((movie) => (
```

Comment:
karena sudah dikasih default value, maka type dari movie adalah array, sehingga gak perlu ?

```suggestion
        movies.map((movie) => (
```